docs/source/migrate_experiment_tracking.md

```diff
@@ -0,0 +1,141 @@
+# Migration from Kedro-Viz Native Experiment Tracking to [`kedro-mlflow`](https://github.com/kedro-org/kedro-viz/pull/2250)
+
+With the deprecation of Kedro-Viz experiment tracking from version 11.0.0, transitioning to [`kedro-mlflow`](https://kedro-mlflow.readthedocs.io/en/stable/) offers enhanced experiment tracking and artifact management. This guide outlines the steps to:
+
+1. **Remove deprecated configurations related to Kedro-Viz experiment tracking.**
+2. **Update existing dataset configurations in the catalog.**
+3. **Optionally, delete old experiment tracking data.**
+4. **Refer to MLflow documentation for further setup.**
+
+## Remove deprecated configurations related to Kedro-Viz experiment tracking
+
+### 1. Remove Session Store Setup
+
+In `src/<project_name>/settings.py`, locate and remove the session store configuration:
+
+```python
+from kedro_viz.integrations.kedro.sqlite_store import SQLiteStore
+
+SESSION_STORE_CLASS = SQLiteStore
+SESSION_STORE_ARGS = {"path": str(Path(__file__).parents[2] / "data")}
+```
+
+### 2. Remove Environment Variables
+Remove any environment variables associated with collaborative tracking:
+
+```bash
+export KEDRO_SQLITE_STORE_USERNAME="your_unique_username"
```

Comment:
This is the command to _set_ the env variable, not remove it. Or do you mean inside the codebase?